providers/implementations/kem/ec_kem.c

```diff
@@ -0,0 +1,842 @@
+/*
+ * Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Shouldn't the name of this source file reference dhkem in some way, e.g. ec_dhkem.c

Reply:
No it shouldnt...
There is a ctx->mode which is set to KEM_MODE_DHKEM..
There could possible be other EC KEM types in future..
This decision was made when we did the RSA KEM... i.e. You have to set OSSL_KEM_PARAM_OPERATION before using it (there is no default).